README.md

```diff
@@ -1,65 +1,105 @@
 # Sonatype Restlet1x
 
-This repository holds an ancient Restlet 1.1.x that is patched here and there for needs of Nexus.
+This repository holds an ancient Restlet 1.1.x that is patched here and there for needs of Sonatype Nexus Repository Manager.
 
-## Releasing it
+## Build A Sonatype / Restlet Release
 
-Is manual process and is a bit labourous. Steps needed (starting from the root of repo):
+There is a script in the same directory as this README.md - `build.xml`. This script is supposed to document and make easier the building of a single zip file containing the dependencies used
+by Sonatype Nexus Repository Manager. It simply calls out to the main restlet build file to create a Maven distribution. Then the script copies only the needed dependencies in Maven 2 layout into a single zip file
+suitable to upload to a Nexus RM content-compressed endpoint.
+
+### Prepare
+
+First make sure you are running the build with Java 7, else you might get random test failures because of how HashMap ordering changed in JDK8.
+
+In other words, make sure `java -version` shows java 7 - the build.xml will also check for this. 
+
+Next, adjust the release number of the build to what you need.
 
 ```
-[cstamas@zaphod restlet1x (master)]$ cd restlet-1.1.6-5346-sonatype/build/
-[cstamas@zaphod build (master)]$ edit build.properties // set release-number to what needed
-[cstamas@zaphod build (master)]$ ant -Dmaven=true
-...
-BUILD SUCCESSFUL
-Total time: 1 minute 30 seconds
-[cstamas@zaphod build (master)]$ cd dist/maven2/restlet-1.1.6-SONATYPE-5348-V8/
-[cstamas@zaphod build (master)]$
+[cstamas@zaphod restlet1x (master)]$ edit restlet-1.1.6-5346-sonatype/build/build.properties
 ```
 
-And you end up in the root of Restlet Maven2 repository. Warning: this repository contains
-all the dependencies too! We need only few (Restlet) artifacts from here, best is to "stage"
-them separately, into new repository. Stage only following artifacts, best into newly
-created empty directory somewhere on your system:
+Look for the property `release-number`. Set this value to your custom release number.
+
+### Build It
+
+Run the build using ant from the same directory as this readme.
+
+```
+> ant
```

Comment:
this ant invocation refers to using the build.xml in the directory of this README.md, which itself sets the maven property to true for you ;)

https://github.com/sonatype/restlet1x/pull/7/files#diff-2cccd7bf48b7a9cc113ff564acd802a8R29